dex/testing/loadbot/mantle.go

```diff
@@ -389,17 +466,20 @@ func coreLimitOrder(sell bool, qty, rate uint64) *core.TradeForm {
 // midGap parses the provided order book for the mid-gap price. If the book
 // is empty, a default value of rateStep * midGapMultiplier is returned instead.
 func midGap(book *core.OrderBook, rateStep uint64) uint64 {
+	if keepMidGap {
+		return uint64(defaultMidGap * 1e8)
+	}
 	if len(book.Sells) > 0 {
 		if len(book.Buys) > 0 {
-			return encodeRate((book.Buys[0].Rate + book.Sells[0].Rate) / 2)
+			return (book.Buys[0].MsgRate + book.Sells[0].MsgRate) / 2
```

Comment:
This difference had me confused for a while. Cold not understand why the mid gap for eth would be so messed up.